src/develop/lightroom.c

```diff
@@ -907,6 +909,78 @@ static void _lrop(const dt_develop_t *dev, const xmlDocPtr doc, const int imgid,
       tcNode = tcNode->next;
     }
   }
+  else if(!xmlStrcmp(name, (const xmlChar *)"title"))
+  {
+	xmlNodePtr ttlNode = node;
+    while(ttlNode)
+    {
+      if(!xmlStrncmp(ttlNode->name, (const xmlChar *)"li", 2))
+      {
+        xmlChar *cvalue = xmlNodeListGetString(doc, ttlNode->xmlChildrenNode, 1);
+		dt_metadata_set(imgid, "Xmp.dc.title", (char *)cvalue);
+        xmlFree(cvalue);
+      }
+      ttlNode = ttlNode->next;
+   }
+  }
+  else if(!xmlStrcmp(name, (const xmlChar *)"description"))
+  {
+	xmlNodePtr desNode = node;
+    while(desNode)
+    {
+      if(!xmlStrncmp(desNode->name, (const xmlChar *)"li", 2))
+      {
+		xmlChar *cvalue = xmlNodeListGetString(doc, desNode->xmlChildrenNode, 1);
+		dt_metadata_set(imgid, "Xmp.dc.description", (char *)cvalue);
+        xmlFree(cvalue);
+      }
+      desNode = desNode->next;
+    }
+  }
+  else if(!xmlStrcmp(name, (const xmlChar *)"creator"))
+  {
+	xmlNodePtr creNode = node;
+    while(creNode)
+    {
+      if(!xmlStrncmp(creNode->name, (const xmlChar *)"li", 2))
+      {
+		xmlChar *cvalue = xmlNodeListGetString(doc, creNode->xmlChildrenNode, 1);
+		dt_metadata_set(imgid, "Xmp.dc.creator", (char *)cvalue);
+        xmlFree(cvalue);
+      }
+      creNode = creNode->next;
+    }
+  }
+  /* Publisher (Dublin Core Schema not supported by Lightroom
```

Comment:
then we do not need this section. just add a clear comment in the commit log that there is no published in lr. or an alternative solution is to have publisher use the creator if present?